readme.md

```diff
@@ -92,6 +92,56 @@ let array = [
 
 ---
 
+#### 📍 no-empty
+disallow empty block statements
+
+##### ❌ Example of incorrect code for this rule:
+
+```js
+if (foo) {
+}
+
+while (foo) {
+}
+
+switch(foo) {
+}
+
+try {
+    doSomething();
+} catch(ex) {
+
+} finally {
+
+}
+```
+
+##### ✅ Example of correct code for this rule:
+
+```js
+if (foo) {
+    // empty
```

Comment:
Is this actually valid? 😱

Reply:
yeah as the comment is classed as content. these were the examples given on eslint. if we find these in each other code though we should definitely  name and shame in a very public way.